exercises/Size-biased-extinction.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: exercise
+title: Size-biased extinction
+subtitle: Putting it all together
```

Comment:
I would think that 'Putting it all together' or 'Capstone Assignment' would be the title. With 'Sized-biased extinction' the subtitle. In general, I've organized titles as a broad topic or concept, with the subtitle describing the specific topic, concept, or story of the exercise.